views/js/controller/creator/creator.js

```diff
@@ -74,6 +74,23 @@ define([
     ('use strict');
     const logger = loggerFactory('taoQtiTest/controller/creator');
 
+    /**
+     * We assume the ClientLibConfigRegistry is filled up with something like this:
+     * 'taoQtiTest/controller/creator/creator' => [
+     *     'provider' => 'qtiTest',
+     * ],
+     *
+     * Or, with something like this for allowing multiple buttons in case of several providers are available:
+     * 'taoQtiTest/controller/creator/creator' => [
+     *     'provider' => 'qtiTest',
+     *     'providers' => [
+     *         ['id' => 'qtiTest', 'label' => 'Preview'],
+     *         ['id' => 'xxxx', 'label' => 'xxxx'],
+     *         ...
+     *     ],
+     * ],
+     */
```

Comment:
I've mentioned it in the other PR about items. This should be in a private method `getProviders` JSDoc describing the contract and helping even IDE autocomplete and documenting the code (since we do not have domain objects for this).